shared/unattended/Fedora-25.ks

```diff
@@ -34,15 +33,17 @@ function ECHO { for TTY in `cat /proc/consoles | awk '{print $NF}'`; do source "
 ECHO "OS install is completed"
 grubby --remove-args="rhgb quiet" --update-kernel=$(grubby --default-kernel)
 dhclient
-chkconfig sshd on
 iptables -F
 systemctl mask tmp.mount
-echo 0 > /selinux/enforce
+selinux --enforcing
 sed -i "/^HWADDR/d" /etc/sysconfig/network-scripts/ifcfg-eth0
-# if package groups were missing from main installation repo
+# if packages were missing from main installation repo
 # try again from installed system
 dnf -y groupinstall c-development development-tools
-# include avocado: allows using this machine with remote runner
-dnf -y install python2-avocado
+dnf -y install net-tools sg3_utils python3-avocado
```

Comment:
Oh, that is true, I always forget some pkgs are not available directly in installation but they are available post-install...